opentok/opentok-meet
# OPENTOK-44296: Log if p2p4121 is enabled in meet

#### What is this PR doing?
In the publisher stats widget, logs the current media mode in use and whether or not adaptive media routing is enabled at all. Also labels the Subscriber Id.

#### How should this be manually tested?
 1. Without p2p4121 enable (such as with the apikey set to "100"), publish in a mantis session. 
 2. Verify that the "media mode" and "Adaptive Media Routing" fields are correct. 
 3. Do the same in a p2p room (you'll need to add a second participant) and also verify that the subscriberId is present. 
-------------------
1. With p2p4121 enabled (such as with the apikey set to "1302"), join a room with only one participant. 
2. Verify that the media mode is mantis and the adaptive media routing field says it's enabled. 
3. Join as a second participant and verify that the media mode has transitioned to p2p and the subscriberId is present. 
4. Join as a third participant and verify that the media mode has transitioned back to mantis.

#### What are the relevant tickets?
https://jira.vonage.com/browse/OPENTOK-44296
